src/odemis/cli/main.py

```diff
@@ -331,6 +332,32 @@ def print_vattribute(name, va, pretty):
                 logging.info("Failed to convert %s to component names")
                 # Leave the value as-is
 
+        # Convert to nicer unit for user
+        if va.unit and va.unit == "rad" and isinstance(val, numbers.Real):
+            try:
+                val_converted = u" = %s°" % (math.degrees(val),)
```

Comment:
I'm wondering. Why actually do you make it unicode? Why not string?

Reply:
It's only for Python 2. In Python 3, all strings are unicode. We don't really need this anymore, as we now only support Python 3, but to follow the style of the rest of the code, I keep unicode here too. One day, someone will convert this code to Python 3 only, but that day is not today ;-)